tycho-eclipse-plugin/src/main/java/org/eclipse/tycho/eclipserun/EclipseRunMojo.java

```diff
@@ -105,7 +105,7 @@ public class EclipseRunMojo extends AbstractMojo {
 	@Parameter(defaultValue = "true")
 	private boolean clearWorkspaceBeforeLaunch;
 
-	@Parameter(property = "project")
+	@Parameter(property = "project", readonly = true)
```

Comment:
```suggestion
	@Component
```

Is actually the Maven3 way even though it is claimed to be deperated.

The next (JSR330) way should also work


```suggestion
	@Inject
```